### ISSUE TITLE
[docs] React Native 0.71.* integration

### ISSUE THREAD

**Issue by whalemare** [zoontek/react-native-bootsplash#421]
### Why it is needed?

In new react-native 0.71.* have been change implementation of AppDelegate.mm to simplify future upgrade process, that hide access to rootView under the hood:

```
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions 
{
  self.moduleName = @"app"
  return [super application:application didFinishLaunchingWithOptions:launchOptions];
}
```

But with react-native-bootsplash we need to get access to `rootView`:
```
RNBootSplash initWithStoryboard:@"BootSplash" rootView:rootView];
```

### Possible implementation

I am not sure about best-practice in Objective-C, but this solution worked: 

```
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions 
{
  self.moduleName = @"app";
  [super application:application didFinishLaunchingWithOptions:launchOptions];
  [RNBootSplash initWithStoryboard:@"BootSplash" rootView:self.window.rootViewController.view];
  return YES;
}
```

**Comment by zoontek** [comment]
@whalemare Yes, that's correct. I will update the README when the stable version will be available.

**Comment by efstathiosntonas** [comment]
@zoontek heads up, 0.71 just rolled out the shop! :)

**Comment by rililive** [comment]
Thanks! This comment helped unblock us!

**Comment by zoontek** [comment]
I published [version `4.4.1`](https://github.com/zoontek/react-native-bootsplash/releases/tag/4.4.1) with an updated documentation and example app.
I went for the "link to previous README" option since it's allowing me to remove the Android setup first step.

<img width="539" alt="Screenshot 2023-01-16 at 12 32 20" src="https://user-images.githubusercontent.com/1902323/212668814-990672ab-a33b-4592-ae62-2d358207ea4e.png">
<img width="870" alt="Screenshot 2023-01-16 at 12 32 40" src="https://user-images.githubusercontent.com/1902323/212668830-7841c652-0a8e-4b21-996a-75191f66cb7b.png">
